### ISSUE TITLE
 test_tpm2_nv.sh: fix errors during test

### ISSUE THREAD

**Issue by williamcroberts** [01org/tpm2-tools#386]
```
70 6c 65 61 73 65 31 32 33 61 62 63 0a 

000000: be be be be be be be be be be be be be be be be  ................
000010: be be be be be be be be be be be be be be be be  ................
  0. NV Index: 0x1500018
release the nv index OK!
head: invalid number of bytes: ‘0x00000800’
ERROR: Failed to read NVRAM area at index 0x1000000 (16777216). Error:0x14a
  0. NV Index: 0x1000000
 test_tpm2_nv.sh pass 
```

**Comment by martinezjavier** [comment]
@williamcroberts this should had been fixed by commit b410e7f4a4a0 ("test: fix tpm2_nv.sh passing a hex number as head command -c argument").

I didn't notice that there was an issue already filled when I created issue #463. Sorry about that.

**Comment by williamcroberts** [comment]
closed via #463
